File: Server/Classes/Packet.js
class Packet {
    type;
    event;
    value;

    constructor(type, event, value) {
        this.type = type;
        this.event = event;
        this.value = value;
    }

    asPacket() {
        return JSON.stringify({
            t : this.type,
            e : this.event,
            v : this.value
        });
    }
}

File: DragonFlightClient/Assets/01. Scripts/Server/VO/Packet.cs
using Newtonsoft.Json;

public struct Packet
{
    [JsonProperty("t")] public int type;
    [JsonProperty("e")] public int eventType;
    [JsonProperty("v")] public string value;

    public Packet(int type, int eventType, string value)
    {
        this.type = type;
        this.eventType = eventType;
        this.value = value;
    }
}
